util/steambox

```diff
@@ -51,9 +51,12 @@ declare -a DRIDEVS=()
 if [[ -d /dev/dri ]] ; then
 	DRIDEVS+=(/dev/dri/*)
 fi
-if [[ -d /dev/ati ]] ; then 
+if [[ -d /dev/ati ]] ; then
 	DRIDEVS+=(/dev/ati/*)
 fi
+if [[ -e /dev/nvidiactl ]] ; then
+	DRIDEVS+=(/dev/nvidia*)
```

Comment:
@jcorrado The `if` statement checks for nvidiactl but the `DRIDEVS` line pulls in /dev/nvidia*.   Was this intentional?

I don't know what the nvidia driver directory tree looks like so I'll take your word for it if that's how it should be.

Note that another method of building this array would be something like:
```bash
   DRIDEVS += ( $(find /dev -name nvidia* -type c -print) )
```
This line would pull in all the "character devices" named `nvidia*` in the dev tree.

I can't say which is better or worse... the "find" line will only pull character devices where the original line will pull character devices, block devices, and possibly symbolic links.